data/sessions.json

```diff
@@ -0,0 +1,51 @@
+[
+    {
+        "idSession": 1,
```

Comment:
since the name of the entity is `Session` is not necessary clarify it in the `id` name, just use `id` instead of `idSession`